global-api/routes/city_locode_endpoint_ghgrp.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+from fastapi import APIRouter, HTTPException
+import math
+from sqlalchemy import text
+import pandas as pd
+from db.database import SessionLocal
+
+api_router = APIRouter(prefix="/api/v0")
+
+gpc_quality_data = "TBD"
+gpc_quality_EF = "TBD"
```

Comment:
I think we need to figure these out before merging the PR, correct?

Reply:
yes